plugins/modules/vmadm.py

```diff
@@ -361,7 +361,7 @@ def get_vm_prop(module, uuid, prop):
     (rc, stdout, stderr) = module.run_command(cmd)
 
     if rc != 0:
-        module.fail_json(msg=f"Could not perform lookup of {prop} on {uuid}", exception=stderr)
+        module.fail_json(msg=f"Could not perform lookup of {prop} on {uuid}:\n{stderr}")
```

Comment:
The more correct solution would be to return stderr as stderr. And while we're at it, also return the other information on the command that was run:
```suggestion
        module.fail_json(msg=f"Could not perform lookup of {prop} on {uuid}", command=cmd, stderr=stderr, stdout=stdout, rc=rc)
```